components/template/template-application-rest/src/main/resources/META-INF/dirigible/template-application-rest/template/template.js

```diff
@@ -57,12 +57,19 @@ export function getTemplate(parameters) {
         rename: "tsconfig.json",
         engine: "velocity"
     }];
+
+    templateSources.push({
+        location: "/template-application-rest/roles/roles.roles.template",
```

Comment:
You could rename the template name to `default-roles.roles.template`, so it matches the generated file name :)